### PR TITLE
docs: using zk rust with aligned

### LINKED ISSUE #590: docs: use zk rust with aligned

### PR DESCRIPTION
**Pull request by taturosati**
Docs on how to use zk rust to generate and submit a proof to aligned

### CODE REVIEW

File: docs/guides/0_5_using_zkrust.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+cast wallet import --interactive <path_to_keystore.json>
+```
+
+Make sure to send at least 0.1 ETH to the address in the keystore.
+You can get holesky eth from the [faucet](https://cloud.google.com/application/web3/faucet/ethereum/holesky)
+
+2. Generate and submit the proof:
+
+```bash
+cargo run --release -- prove-sp1 \
```
Comment (JuArce) [inline docs/guides/0_5_using_zkrust.md:59]: We are mostly using make to run the commands. Maybe this example should use make

File: docs/SUMMARY.md [hunk 2]
```diff
@@ -26,6 +26,7 @@
 ## Guides
 
 * [Submitting proofs](guides/0_submitting_proofs.md)
+* [Generating & Submiting Proofs with ZKRust](guides/0_5_using_zkrust.md)
```
Comment (JuArce) [inline docs/SUMMARY.md]:
```suggestion
* [Generating & Submitting Proofs with ZKRust](guides/0_5_using_zkrust.md)
```

File: docs/guides/0_5_using_zkrust.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Generating & Submiting proofs to Aligned using ZKRust
+
+## Dependencies
+
+To generate and submit proofs to Aligned using ZKRust, you need to have the following dependencies installed:
+- [Rust](https://www.rust-lang.org/tools/install)
+- [Foundry](https://book.getfoundry.sh/getting-started/installation)
+- [ZKRust](https://github.com/lambdaclass/zkRust)
```
Comment (JuArce) [inline docs/guides/0_5_using_zkrust.md]: It is not clear. zkrust is not an installable

File: docs/guides/0_5_using_zkrust.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Make sure to send at least 0.1 ETH to the address in the keystore.
+You can get holesky eth from the [faucet](https://cloud.google.com/application/web3/faucet/ethereum/holesky)
+
+2. Generate and submit the proof:
+
+```bash
+cargo run --release -- prove-sp1 \
+    --submit-to-aligned-with-keystore <path_to_keystore> \
+    examples/fibonacci_no_std .
```
Comment (JuArce) [inline docs/guides/0_5_using_zkrust.md]:
```suggestion
    examples/fibonacci .
```

**Review by JuArce — CHANGES_REQUESTED** [review]

File: docs/guides/0_5_using_zkrust.md (hunk 4 above)
Comment (taturosati) [reply docs/guides/0_5_using_zkrust.md]: fixed

**Review by JuArce — APPROVED** [review]

**Review by entropidelic — APPROVED** [review]

**Pull request merged**